modules/models/user.php

```diff
@@ -9,12 +9,30 @@ class User extends Record
     public $id;
     public $student_id;
     public $first_name;
-
     public $last_name;
     public $email;
     public $is_admin;
     public $is_instructor;
     public $password_digest;
     public $created_at;
     public $updated_at;
+
+    public function get_possible_roles()
+    {
+        $res = array();
+
+        if ($this->is_admin) {
+            array_push($res, "admin");
+        }
+
+        if ($this->is_instructor) {
+            array_push($res, "instructor");
+        }
+
+        if (isset($this->student_id)) {
+            array_push($res, "student");
+        }
+
```

Comment:
Should we add TA here as well?

Reply:
@drouba do we have a way of telling if someone is a TA? I don't see it